### DISCUSSION TITLE
Customer Previous Runs

### DISCUSSION THREAD

**Discussion by GEORGE-BOAST** [open-meteo/open-meteo#1072] [Q&A]
Hey, I have wrote my own client around your python api to better fit my needs however on the previous runs model, I only get one response, even when querying multiple models. How can I know what model is the response, eg right now I just use the response.,Model() from your sdk https://github.com/open-meteo/sdk/tree/main/python/openmeteo_sdk. 

Another question, when using the previous runs, how can I tell what run is what day, eg I pull all the following info 
![image](https://github.com/user-attachments/assets/7f53fe92-5688-4d5a-8bf9-a0af18c0a14d)
but there doesnt seem to be anything in the sdk that allows me to run what day the run was for?

**Comment by patrick-zippenfenig** [comment]
Hi, yes, you can use `.Model()` to retrieve the model name. Just keep in mind that each model's data will come as a separate response message. You'll need to iterate through all the `responses`. Here is some code to resolve the model name: https://github.com/open-meteo/sdk?tab=readme-ov-file#model
